### ISSUE TITLE
PyTorch: note that need torch.device('cuda') or similar to use GPUs?

### ISSUE THREAD

**Issue by willfurnass** [rcgsheffield/sheffield_hpc#1053]
@twinkarma Think this is worth mentioning in our docs and/or should we point people at https://pytorch.org/docs/stable/notes/cuda.html?

cc @adityagilra

**Comment by twinkarma** [comment]
@willfurnass Yes we should. If you're not in a massive rush I can do this after I'm back from leave as I'm busy with writing GPU hackathon docs at the moment.
